pkg/stub/handler_test.go

```diff
@@ -128,6 +128,27 @@ func TestWithBadArch(t *testing.T) {
 	validate(false, err, "architecture bad unsupported", sr, []v1alpha1.SamplesResourceConditionType{v1alpha1.SamplesExist}, []corev1.ConditionStatus{corev1.ConditionUnknown}, t)
 }
 
+func TestConfigurationValidCondition(t *testing.T) {
+	h, sr, event := setup()
+	err := h.Handle(nil, event)
+	validate(true, err, "", sr, []v1alpha1.SamplesResourceConditionType{v1alpha1.SamplesExist}, []corev1.ConditionStatus{corev1.ConditionTrue}, t)
+	_, sr, event = setup()
+	sr.Spec.InstallType = "rhel8"
+	sr.ResourceVersion = "2"
+	err = h.Handle(nil, event)
+	validate(false, err, "trying to change installtype, which is not allowed, but also specificed", sr, []v1alpha1.SamplesResourceConditionType{v1alpha1.ConfigurationValid}, []corev1.ConditionStatus{corev1.ConditionFalse}, t)
```

Comment:
s/specificed/specified/

Reply:
(hopefully that would have failed your test..)

Reply:
it was misspelled in the src code too ... adjusting in both places